configure.py

```diff
@@ -1051,7 +1051,8 @@ def configure_node(o):
     o['variables']['debug_nghttp2'] = 'false'
 
   o['variables']['node_no_browser_globals'] = b(options.no_browser_globals)
-  o['variables']['node_code_cache_path'] = 'yes'
+  # TODO(refack): fix this when implementing embedded code-cache when cross-compiling.
+  o['variables']['node_code_cache_path'] = 'yes' if o['variables']['want_separate_host_toolset'] == 0 else ''
```

Comment:
Should it not be `no` in the else case?

Reply:
Before https://github.com/nodejs/node/commit/14df42fd008ef8e95d60d0d70084943d180bab91#diff-ade1d3e4b7c35655f854151d899df62b
is was conditionally the path to the code-cache file.
So you are right that it need to change, but IMO it just should not exist.